### ISSUE TITLE
2018-06-23-12-46-26_0.bag 在哪？

### ISSUE THREAD

**Issue by LukyBird** [tony1098/Stereo-Localization#1]

**Comment by tony1098** [comment]
Sorry for the late response. The dataset has not been made public yet, so I am unable to provide you the bag.

**Comment by ZeyingXuHuaWei** [comment]
@tony1098  can we use kitti dataset to do expriment?

**Comment by tony1098** [comment]
@ZeyingXuHuaWei Yes, KITTI should work. But you may have to construct the LIDAR map on your own.
